### PR TITLE
Update fr_FR.json

### PR DESCRIPTION
**Pull request by Pedrobordeaux**

### CODE REVIEW

File: locales/fr_FR.json [hunk 1]
```diff
@@ -1,19 +1,19 @@
 {
   "sandbox-warning": "This is a sandbox environment for testing API integration. Never send real funds here!",
```
Comment (Marcpepe) [inline locales/fr_FR.json:2]: "Ceci est un environnement sandbox destiné aux tests d'intégration de l'API. N'envoyez jamais de fonds réels ici !"

File: locales/fr_FR.json [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-  "and": "and",
-  "from": "From",
-  "to": "To",
-  "or": "or",
-  "price": "Price",
-  "fee": "Fee",
+  "back": "Retour",
+  "help": "Aide",
+  "and": "et",
+  "from": "de",
```
Comment (Marcpepe) [inline locales/fr_FR.json:10]: Depuis

File: locales/fr_FR.json [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
+  "to": "à",
```
Comment (Marcpepe) [inline locales/fr_FR.json:11]: Vers

File: locales/fr_FR.json [hunk 4]
```diff
@@ -1,19 +1,19 @@
...
+  "or": "ou",
+  "price": "Prix",
+  "fee": "Frais",
   "try-again": "Try again",
```
Comment (Marcpepe) [inline locales/fr_FR.json:15]: Réessayer

File: locales/fr_FR.json [hunk 5]
```diff
@@ -1,19 +1,19 @@
...
-  "refresh": "Refresh",
+  "refresh": "Rafraîchir",
   "finish": "Finish",
```
Comment (Marcpepe) [inline locales/fr_FR.json:17]: Terminer

**Comment by canesin** [comment]
@Pedrobordeaux why you closed the PR? I was going to accept it.

**Comment by Pedrobordeaux** [comment]
Sorry I created the project this morning. I have no skills into coding and
this is the first time I use github. I am still trying to figure out how it
works. I didn’t want to make mistake in the way I generate the project so I
wanted to do it again in a proper way that why I closed it. But I will do a
new project.

**Pull request closed without merging**